### ISSUE TITLE
Změna barevného tématu

### ISSUE THREAD

**Issue by milanjirkovsky** [lonebits/tiptrips#6]
Systém mám nastavenej na tmavý schéma (protože světlý notifikace vypadají strašně), ale appky mám všechny přepnutý na bílý schéma, protože černý není na slunci vidět. Bylo by fajn, kdyby to šlo v appce změnit. (low prio)

**Comment by bendiscz** [comment]
Jo, souhlas, tohle by mela byt trivka. Udelam tam nejake settings...
